boards/others/candlelightfd/candlelightfd_stm32g0b1xx.dts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2024 Sean Nyekjaer <sean@geanix.com>
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <st/g0/stm32g0b1Xb.dtsi>
+#include <st/g0/stm32g0b1r(b-c-e)tx-pinctrl.dtsi>
+#include "candlelightfd.dtsi"
```

Comment:
These common includes could be moved from the two DTS files into `candlelightfd.dtsi` to underline that the only difference for the `dual` variant is the mounted CAN transceiver for `fdcan2`.